### PR TITLE
Create more solid Wasm benchmarks

### PR DESCRIPTION
**Pull request by tiziano88**
Ref #4706

### CODE REVIEW

**Comment by tiziano88** [comment]
Current results:

```
   Compiling oak_functions_service v0.1.0 (/home/tzn/src/oak/oak_functions_service)
    Finished bench [optimized] target(s) in 1.50s
     Running benches/wasm_benchmark.rs (target/x86_64-unknown-linux-gnu/release/deps/wasm_benchmark-df9cfec0e4128679)
Gnuplot not found, using plotters backend
    Finished release [optimized] target(s) in 0.06s
echo                    time:   [88.717 µs 89.284 µs 89.960 µs]
                        change: [+0.5912% +1.5240% +2.4785%] (p = 0.00 < 0.05)
                        Change within noise threshold.
Found 6 outliers among 100 measurements (6.00%)
  4 (4.00%) high mild
  2 (2.00%) high severe

    Finished release [optimized] target(s) in 0.07s
lookup                  time:   [107.63 µs 108.03 µs 108.46 µs]
                        change: [+4.1958% +5.2397% +6.4245%] (p = 0.00 < 0.05)
                        Performance has regressed.
Found 3 outliers among 100 measurements (3.00%)
  1 (1.00%) high mild
  2 (2.00%) high severe
```

File: oak_functions_service/benches/wasm_benchmark.rs [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+use oak_functions_abi::Request;
+use oak_functions_service::{
+    logger::StandaloneLogger,
+    lookup::{Data, LookupDataManager},
+    wasm::WasmHandler,
+};
+use rand::{rngs::SmallRng, Rng, SeedableRng};
+
+fn bench_invoke_echo(c: &mut Criterion) {
+    let test_state = create_test_state_with("echo");
```
Comment (ernoc) [inline oak_functions_service/benches/wasm_benchmark.rs:31]: For easier readability, consider naming this `create_test_state_with_name` or adding explicitly `create_test_state_with(/* name= */ "echo")`. Without that, when I first read it I thought the argument was more like a test payload or test data not a test name.

**Review by ernoc — COMMENTED** [review]
LG but probably seek approval for an additional reviewer.

**Review by thmsbinder — APPROVED** [review]

File: oak_functions_service/benches/wasm_benchmark.rs (hunk 1 above)
Comment (tiziano88) [reply oak_functions_service/benches/wasm_benchmark.rs]: Done.

**Pull request merged**